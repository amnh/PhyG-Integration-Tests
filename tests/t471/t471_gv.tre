digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=T0];
    1 [label=T1];
    2 [label=T10];
    3 [label=T11];
    4 [label=T12];
    5 [label=T13];
    6 [label=T14];
    7 [label=T15];
    8 [label=T16];
    9 [label=T17];
    10 [label=T18];
    11 [label=T19];
    12 [label=T2];
    13 [label=T20];
    14 [label=T21];
    15 [label=T22];
    16 [label=T23];
    17 [label=T24];
    18 [label=T3];
    19 [label=T4];
    20 [label=T5];
    21 [label=T6];
    22 [label=T7];
    23 [label=T8];
    24 [label=T9];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    37 [label=HTU37];
    38 [label=HTU38];
    39 [label=HTU39];
    40 [label=HTU40];
    41 [label=HTU41];
    42 [label=HTU42];
    43 [label=HTU43];
    44 [label=HTU44];
    45 [label=HTU45];
    46 [label=HTU46];
    47 [label=HTU47];
    48 [label=HTU48];
    49 [label=HTU49];
    50 [label=HTU50];
    51 [label=HTU51];
    52 [label=HTU52];
    25 -> 0 [label=0.0];
    25 -> 26 [label=1.0];
    26 -> 27 [label=88.0];
    26 -> 34 [label=71.0];
    27 -> 9 [label=72.0];
    27 -> 28 [label=56.0];
    28 -> 14 [label=97.0];
    28 -> 29 [label=90.0];
    29 -> 3 [label=86.0];
    29 -> 30 [label=47.0];
    30 -> 31 [label=33.0];
    30 -> 33 [label=0.0];
    31 -> 8 [label=41.0];
    31 -> 32 [label=0.0];
    32 -> 1 [label=87.0];
    33 -> 19 [label=94.0];
    34 -> 35 [label=75.0];
    34 -> 51 [label=26.0];
    35 -> 21 [label=84.0];
    35 -> 36 [label=74.0];
    36 -> 10 [label=85.0];
    36 -> 37 [label=53.0];
    37 -> 38 [label=69.0];
    37 -> 47 [label=87.0];
    38 -> 23 [label=81.0];
    38 -> 39 [label=70.0];
    39 -> 12 [label=68.0];
    39 -> 40 [label=72.0];
    40 -> 24 [label=72.0];
    40 -> 41 [label=76.0];
    41 -> 22 [label=66.0];
    41 -> 42 [label=63.0];
    42 -> 43 [label=82.0];
    42 -> 46 [label=70.0];
    43 -> 11 [label=66.0];
    43 -> 44 [label=84.0];
    44 -> 2 [label=70.0];
    44 -> 45 [label=36.0];
    45 -> 13 [label=95.0];
    45 -> 32 [label=0.0];
    46 -> 7 [label=89.0];
    46 -> 17 [label=74.0];
    47 -> 48 [label=83.0];
    47 -> 50 [label=76.0];
    48 -> 6 [label=78.0];
    48 -> 49 [label=70.0];
    49 -> 5 [label=87.0];
    49 -> 20 [label=76.0];
    50 -> 15 [label=90.0];
    50 -> 16 [label=68.0];
    51 -> 33 [label=0.0];
    51 -> 52 [label=83.0];
    52 -> 4 [label=74.0];
    52 -> 18 [label=88.0];
}
//4605.5625
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=T0];
    1 [label=T1];
    2 [label=T10];
    3 [label=T11];
    4 [label=T12];
    5 [label=T13];
    6 [label=T14];
    7 [label=T15];
    8 [label=T16];
    9 [label=T17];
    10 [label=T18];
    11 [label=T19];
    12 [label=T2];
    13 [label=T20];
    14 [label=T21];
    15 [label=T22];
    16 [label=T23];
    17 [label=T24];
    18 [label=T3];
    19 [label=T4];
    20 [label=T5];
    21 [label=T6];
    22 [label=T7];
    23 [label=T8];
    24 [label=T9];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    37 [label=HTU37];
    38 [label=HTU38];
    39 [label=HTU39];
    40 [label=HTU40];
    41 [label=HTU41];
    42 [label=HTU42];
    43 [label=HTU43];
    44 [label=HTU44];
    45 [label=HTU45];
    46 [label=HTU46];
    47 [label=HTU47];
    48 [label=HTU48];
    49 [label=HTU49];
    50 [label=HTU50];
    51 [label=HTU51];
    52 [label=HTU52];
    25 -> 0 [label=0.0];
    25 -> 26 [label=1.0];
    26 -> 27 [label=70.0];
    26 -> 48 [label=87.0];
    27 -> 28 [label=75.0];
    27 -> 45 [label=26.0];
    28 -> 21 [label=84.0];
    28 -> 29 [label=74.0];
    29 -> 10 [label=85.0];
    29 -> 30 [label=53.0];
    30 -> 31 [label=69.0];
    30 -> 41 [label=87.0];
    31 -> 23 [label=81.0];
    31 -> 32 [label=70.0];
    32 -> 12 [label=68.0];
    32 -> 33 [label=72.0];
    33 -> 24 [label=72.0];
    33 -> 34 [label=76.0];
    34 -> 22 [label=66.0];
    34 -> 35 [label=63.0];
    35 -> 36 [label=82.0];
    35 -> 40 [label=70.0];
    36 -> 11 [label=66.0];
    36 -> 37 [label=84.0];
    37 -> 2 [label=70.0];
    37 -> 38 [label=36.0];
    38 -> 13 [label=95.0];
    38 -> 39 [label=0.0];
    39 -> 1 [label=87.0];
    40 -> 7 [label=89.0];
    40 -> 17 [label=74.0];
    41 -> 42 [label=83.0];
    41 -> 44 [label=76.0];
    42 -> 6 [label=78.0];
    42 -> 43 [label=70.0];
    43 -> 5 [label=87.0];
    43 -> 20 [label=76.0];
    44 -> 15 [label=90.0];
    44 -> 16 [label=68.0];
    45 -> 46 [label=83.0];
    45 -> 47 [label=0.0];
    46 -> 4 [label=74.0];
    46 -> 18 [label=88.0];
    47 -> 19 [label=42.0];
    48 -> 9 [label=72.0];
    48 -> 49 [label=56.0];
    49 -> 14 [label=96.0];
    49 -> 50 [label=90.0];
    50 -> 3 [label=86.0];
    50 -> 51 [label=47.0];
    51 -> 47 [label=0.0];
    51 -> 52 [label=33.0];
    52 -> 8 [label=77.0];
    52 -> 39 [label=0.0];
}
//4605.5625
