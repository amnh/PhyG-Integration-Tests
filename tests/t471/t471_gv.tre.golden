digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=0.0];
    17 -> 18 [label=13.9];
    17 -> 27 [label=7.0];
    18 -> 19 [label=56.1];
    18 -> 24 [label=48.0];
    19 -> 20 [label=60.0];
    19 -> 22 [label=113.0];
    20 -> 1 [label=174.2];
    20 -> 21 [label=95.0];
    21 -> 3 [label=72.7];
    21 -> 13 [label=46.0];
    22 -> 15 [label=111.2];
    22 -> 36 [label=0.0];
    23 -> 8 [label=195.3];
    23 -> 10 [label=130.0];
    24 -> 25 [label=26.1];
    24 -> 31 [label=37.8];
    25 -> 14 [label=91.6];
    25 -> 26 [label=13.0];
    26 -> 6 [label=52.7];
    26 -> 34 [label=0.0];
    27 -> 9 [label=78.4];
    27 -> 32 [label=0.0];
    28 -> 11 [label=55.2];
    28 -> 29 [label=24.0];
    29 -> 12 [label=119.3];
    29 -> 30 [label=45.0];
    30 -> 4 [label=104.6];
    30 -> 5 [label=96.9];
    31 -> 32 [label=0.0];
    31 -> 35 [label=0.0];
    32 -> 28 [label=45.6];
    33 -> 2 [label=177.1];
    33 -> 34 [label=0.0];
    34 -> 7 [label=101.3];
    35 -> 33 [label=35.2];
    35 -> 36 [label=0.0];
    36 -> 23 [label=63.7];
}
//3796.1
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=0.0];
    17 -> 2 [label=122.0];
    17 -> 18 [label=59.0];
    18 -> 19 [label=21.0];
    18 -> 27 [label=22.0];
    19 -> 25 [label=42.1];
    19 -> 31 [label=44.1];
    20 -> 21 [label=100.0];
    20 -> 23 [label=66.0];
    21 -> 15 [label=113.2];
    21 -> 22 [label=61.7];
    22 -> 8 [label=196.3];
    22 -> 10 [label=134.0];
    23 -> 1 [label=172.2];
    23 -> 24 [label=103.0];
    24 -> 3 [label=68.7];
    24 -> 13 [label=48.0];
    25 -> 14 [label=72.0];
    25 -> 26 [label=20.0];
    26 -> 7 [label=79.0];
    26 -> 34 [label=0.0];
    27 -> 9 [label=69.5];
    27 -> 32 [label=0.0];
    28 -> 5 [label=66.0];
    28 -> 36 [label=0.0];
    29 -> 12 [label=53.3];
    29 -> 30 [label=48.1];
    30 -> 4 [label=70.5];
    30 -> 11 [label=60.1];
    31 -> 32 [label=0.0];
    31 -> 33 [label=11.0];
    32 -> 28 [label=37.0];
    33 -> 34 [label=0.0];
    33 -> 35 [label=23.1];
    34 -> 6 [label=72.9];
    35 -> 20 [label=49.9];
    35 -> 36 [label=0.0];
    36 -> 29 [label=92.8];
}
//3697.6
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=0.0];
    17 -> 18 [label=51.7];
    17 -> 22 [label=39.0];
    18 -> 9 [label=71.5];
    18 -> 34 [label=0.0];
    19 -> 11 [label=41.1];
    19 -> 32 [label=0.0];
    20 -> 12 [label=59.4];
    20 -> 36 [label=0.0];
    21 -> 4 [label=134.0];
    21 -> 5 [label=60.5];
    22 -> 31 [label=0.0];
    22 -> 33 [label=3.0];
    23 -> 24 [label=47.3];
    23 -> 35 [label=48.1];
    24 -> 14 [label=65.0];
    24 -> 25 [label=23.0];
    25 -> 6 [label=90.3];
    25 -> 7 [label=79.0];
    26 -> 27 [label=113.0];
    26 -> 29 [label=47.0];
    27 -> 15 [label=113.2];
    27 -> 28 [label=75.7];
    28 -> 8 [label=195.3];
    28 -> 10 [label=132.0];
    29 -> 1 [label=174.2];
    29 -> 30 [label=101.0];
    30 -> 3 [label=72.7];
    30 -> 13 [label=49.0];
    31 -> 2 [label=153.0];
    31 -> 32 [label=0.0];
    32 -> 20 [label=22.0];
    33 -> 23 [label=75.1];
    33 -> 34 [label=0.0];
    34 -> 19 [label=30.1];
    35 -> 26 [label=63.0];
    35 -> 36 [label=0.0];
    36 -> 21 [label=22.5];
}
//3689.1
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=1.0];
    17 -> 2 [label=118.2];
    17 -> 18 [label=44.0];
    18 -> 19 [label=75.0];
    18 -> 28 [label=70.9];
    19 -> 9 [label=68.7];
    19 -> 34 [label=8.0];
    20 -> 7 [label=116.4];
    20 -> 21 [label=21.0];
    21 -> 22 [label=78.0];
    21 -> 27 [label=18.0];
    22 -> 23 [label=106.0];
    22 -> 25 [label=48.0];
    23 -> 15 [label=105.2];
    23 -> 24 [label=75.7];
    24 -> 10 [label=128.0];
    24 -> 36 [label=0.0];
    25 -> 1 [label=171.2];
    25 -> 26 [label=91.0];
    26 -> 3 [label=72.7];
    26 -> 13 [label=51.0];
    27 -> 6 [label=60.9];
    27 -> 14 [label=93.4];
    28 -> 29 [label=36.0];
    28 -> 30 [label=22.0];
    29 -> 4 [label=77.5];
    29 -> 33 [label=0.0];
    30 -> 32 [label=0.0];
    30 -> 35 [label=0.0];
    31 -> 20 [label=26.9];
    31 -> 32 [label=0.0];
    32 -> 5 [label=76.9];
    33 -> 11 [label=54.1];
    33 -> 34 [label=0.0];
    34 -> 31 [label=5.0];
    35 -> 12 [label=70.4];
    35 -> 36 [label=0.0];
    36 -> 8 [label=193.3];
}
//3629.3
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=0.0];
    17 -> 18 [label=57.0];
    17 -> 27 [label=34.9];
    18 -> 19 [label=31.0];
    18 -> 33 [label=0.0];
    19 -> 32 [label=0.0];
    19 -> 35 [label=28.0];
    20 -> 21 [label=55.0];
    20 -> 23 [label=117.0];
    21 -> 1 [label=184.2];
    21 -> 22 [label=93.0];
    22 -> 3 [label=74.7];
    22 -> 13 [label=47.0];
    23 -> 15 [label=116.2];
    23 -> 24 [label=59.7];
    24 -> 8 [label=199.3];
    24 -> 10 [label=134.0];
    25 -> 14 [label=65.0];
    25 -> 26 [label=26.0];
    26 -> 6 [label=94.3];
    26 -> 7 [label=76.0];
    27 -> 28 [label=76.0];
    27 -> 31 [label=57.1];
    28 -> 12 [label=83.4];
    28 -> 34 [label=0.0];
    29 -> 11 [label=53.1];
    29 -> 30 [label=11.0];
    30 -> 4 [label=73.5];
    30 -> 36 [label=0.0];
    31 -> 9 [label=74.5];
    31 -> 32 [label=0.0];
    32 -> 2 [label=82.0];
    33 -> 25 [label=44.3];
    33 -> 34 [label=0.0];
    34 -> 29 [label=20.0];
    35 -> 20 [label=59.1];
    35 -> 36 [label=0.0];
    36 -> 5 [label=98.9];
}
//3618.3
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=0.0];
    17 -> 18 [label=32.0];
    17 -> 22 [label=56.0];
    18 -> 31 [label=15.9];
    18 -> 34 [label=0.0];
    19 -> 5 [label=72.9];
    19 -> 36 [label=0.0];
    20 -> 12 [label=65.4];
    20 -> 21 [label=35.0];
    21 -> 4 [label=71.5];
    21 -> 11 [label=61.1];
    22 -> 29 [label=45.2];
    22 -> 33 [label=48.1];
    23 -> 24 [label=59.1];
    23 -> 32 [label=0.0];
    24 -> 25 [label=55.0];
    24 -> 27 [label=113.0];
    25 -> 1 [label=180.2];
    25 -> 26 [label=93.0];
    26 -> 3 [label=74.7];
    26 -> 13 [label=46.0];
    27 -> 15 [label=116.2];
    27 -> 28 [label=58.7];
    28 -> 8 [label=199.3];
    28 -> 10 [label=134.0];
    29 -> 14 [label=66.0];
    29 -> 30 [label=20.0];
    30 -> 6 [label=92.3];
    30 -> 7 [label=75.0];
    31 -> 32 [label=0.0];
    31 -> 35 [label=45.0];
    32 -> 2 [label=146.0];
    33 -> 23 [label=11.0];
    33 -> 34 [label=0.0];
    34 -> 19 [label=36.0];
    35 -> 9 [label=65.5];
    35 -> 36 [label=0.0];
    36 -> 20 [label=15.1];
}
//3612.4
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=1.0];
    17 -> 18 [label=8.0];
    17 -> 27 [label=45.4];
    18 -> 32 [label=0.0];
    18 -> 35 [label=1.0];
    19 -> 20 [label=68.3];
    19 -> 22 [label=80.1];
    20 -> 14 [label=66.0];
    20 -> 21 [label=25.0];
    21 -> 6 [label=93.3];
    21 -> 7 [label=76.0];
    22 -> 23 [label=49.0];
    22 -> 25 [label=111.0];
    23 -> 1 [label=177.2];
    23 -> 24 [label=102.0];
    24 -> 3 [label=72.7];
    24 -> 13 [label=48.0];
    25 -> 15 [label=112.2];
    25 -> 26 [label=67.7];
    26 -> 8 [label=196.3];
    26 -> 10 [label=132.0];
    27 -> 9 [label=65.5];
    27 -> 28 [label=45.2];
    28 -> 30 [label=63.0];
    28 -> 31 [label=66.9];
    29 -> 5 [label=65.0];
    29 -> 34 [label=0.0];
    30 -> 11 [label=64.2];
    30 -> 36 [label=0.0];
    31 -> 29 [label=4.0];
    31 -> 32 [label=7.0];
    32 -> 19 [label=19.9];
    33 -> 2 [label=99.0];
    33 -> 34 [label=0.0];
    34 -> 4 [label=117.0];
    35 -> 33 [label=53.0];
    35 -> 36 [label=0.0];
    36 -> 12 [label=68.3];
}
//3605.9
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=0.0];
    17 -> 18 [label=54.0];
    17 -> 28 [label=55.0];
    18 -> 9 [label=70.6];
    18 -> 31 [label=5.0];
    19 -> 20 [label=28.9];
    19 -> 36 [label=0.0];
    20 -> 21 [label=77.1];
    20 -> 26 [label=51.9];
    21 -> 22 [label=47.0];
    21 -> 24 [label=102.0];
    22 -> 1 [label=172.2];
    22 -> 23 [label=96.0];
    23 -> 3 [label=70.7];
    23 -> 13 [label=49.0];
    24 -> 15 [label=107.2];
    24 -> 25 [label=82.7];
    25 -> 8 [label=195.3];
    25 -> 10 [label=132.0];
    26 -> 6 [label=49.9];
    26 -> 27 [label=38.4];
    27 -> 7 [label=76.0];
    27 -> 14 [label=90.0];
    28 -> 11 [label=55.1];
    28 -> 29 [label=22.0];
    29 -> 32 [label=0.0];
    29 -> 35 [label=4.0];
    30 -> 5 [label=69.0];
    30 -> 34 [label=0.0];
    31 -> 19 [label=20.0];
    31 -> 32 [label=0.0];
    32 -> 30 [label=5.0];
    33 -> 2 [label=112.0];
    33 -> 34 [label=0.0];
    34 -> 4 [label=109.0];
    35 -> 12 [label=96.4];
    35 -> 36 [label=0.0];
    36 -> 33 [label=60.5];
}
//3601.9
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=0.0];
    17 -> 18 [label=16.0];
    17 -> 22 [label=85.0];
    18 -> 31 [label=15.2];
    18 -> 34 [label=0.0];
    19 -> 5 [label=73.0];
    19 -> 36 [label=0.0];
    20 -> 12 [label=69.4];
    20 -> 21 [label=37.0];
    21 -> 4 [label=72.5];
    21 -> 11 [label=55.1];
    22 -> 29 [label=36.0];
    22 -> 33 [label=45.9];
    23 -> 24 [label=67.0];
    23 -> 32 [label=0.0];
    24 -> 25 [label=55.0];
    24 -> 27 [label=115.0];
    25 -> 1 [label=175.2];
    25 -> 26 [label=96.0];
    26 -> 3 [label=71.7];
    26 -> 13 [label=45.0];
    27 -> 15 [label=112.2];
    27 -> 28 [label=61.7];
    28 -> 8 [label=199.3];
    28 -> 10 [label=134.0];
    29 -> 6 [label=49.0];
    29 -> 30 [label=45.3];
    30 -> 7 [label=78.0];
    30 -> 14 [label=92.0];
    31 -> 32 [label=0.0];
    31 -> 35 [label=45.0];
    32 -> 2 [label=140.0];
    33 -> 23 [label=17.0];
    33 -> 34 [label=0.0];
    34 -> 19 [label=34.0];
    35 -> 9 [label=80.5];
    35 -> 36 [label=0.0];
    36 -> 20 [label=15.1];
}
//3600.0
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 17 [label=1.0];
    17 -> 18 [label=10.0];
    17 -> 27 [label=50.4];
    18 -> 32 [label=0.0];
    18 -> 33 [label=43.0];
    19 -> 20 [label=54.9];
    19 -> 22 [label=81.1];
    20 -> 6 [label=52.9];
    20 -> 21 [label=41.4];
    21 -> 7 [label=81.0];
    21 -> 14 [label=91.0];
    22 -> 23 [label=102.0];
    22 -> 25 [label=47.0];
    23 -> 15 [label=114.2];
    23 -> 24 [label=80.7];
    24 -> 8 [label=199.3];
    24 -> 10 [label=134.0];
    25 -> 1 [label=177.2];
    25 -> 26 [label=97.0];
    26 -> 3 [label=72.7];
    26 -> 13 [label=49.0];
    27 -> 9 [label=76.5];
    27 -> 28 [label=35.2];
    28 -> 29 [label=29.0];
    28 -> 31 [label=68.9];
    29 -> 30 [label=54.0];
    29 -> 34 [label=0.0];
    30 -> 11 [label=41.2];
    30 -> 36 [label=0.0];
    31 -> 32 [label=5.0];
    31 -> 35 [label=23.0];
    32 -> 19 [label=26.9];
    33 -> 2 [label=98.0];
    33 -> 34 [label=0.0];
    34 -> 4 [label=91.0];
    35 -> 5 [label=66.0];
    35 -> 36 [label=0.0];
    36 -> 12 [label=69.3];
}
//3577.5
